docs/AIR.md

```diff
@@ -0,0 +1,181 @@
+## AIR: values
```

Comment:
For me, it would be better to know first about instruction and then about values. Because it's really hard to imagine how to deal with values while reading about them, but it's possible to imagine values while read about instructions